### ISSUE TITLE
Platformio-ide-terminal-not-working-in-atom-1.45.0

### ISSUE THREAD

**Issue by Rebevg** [platformio/platformio-atom-ide-terminal#850]
- [X ] I have reset Atom to defaults prior to submitting report.
- [ ] I have not reset Atom to defaults prior to submitting report.
### Description

Platformio-ide-terminal-not-working-in-Atom-1.45.0
When opening the + in Atom no information is showing, not able to write or perform any action. Please see image.
### Steps to reproduce

1. Atom and Platformio-ide-terminal installed
2. Opened a new file and save it
3.Tried to perform Platformio and nothing happens

**Expected behavior:** 
N/A

**Actual behavior:** 
N/A

**Reproduces how often:** 
N/A

### Versions

```
> atom --version
1.45.0
```
```
> apm --version

```
OS name and version:
Platformio-ide-terminal version:
2.10.0

### Additional Information

![image](https://user-images.githubusercontent.com/63929947/79673485-6a331980-8197-11ea-99b1-f86f1eebdeaf.png)
![Platformio](https://user-images.githubusercontent.com/63929947/79673520-9058b980-8197-11ea-849f-cac20b651f6a.JPG)

**Comment by jovancosta-zz** [comment]
same here, with no error code.
the message:
copyright (c) Microsoft Corporation. All rights reserved
(just like your picture)
is not loaded to me.
==
I tried installing it manually, previous versions, and also inside the own atom software and it didn't work.
I'm using windows 10.
Doing the installation manually I got 2 errors. but unfortunately, I've lost these logs. Now it's not appearing any error to me.
Were you able to solve your issue?
==
One thing that I did after installing atom, was:
I uninstalled python 3.7 and installed python 3.8
but I also uninstalled atom and re-installed after it.
and it still not working
